### DISCUSSION TITLE
Search not working in bs5 modal dialog

### DISCUSSION THREAD

**Discussion by sschausten** [brianvoe/slim-select#558] [Q&A] [answered]
Hi, 

I'm using Bootstrap5 modal dialogs for my project and have trouble getting the search function to work. 
I can open my single and multi selects, select or deselect options but I can't use the search. It's there but I can't type/focus.
Slim-select works fine on every other page, just not in the dialogs. 

I would be very grateful if someone had an idea, because the search is very important in the modal dialogs. 
Thanks!

**Comment by brianvoe** [comment, marked as answer]
You can play around with https://slimselectjs.com/settings#contentLocation and https://slimselectjs.com/settings#contentPosition and see if that helps your situation.

But if slim select works and you introduce another framework or your own css and that causes it to break usage of slim select it would probably be best to look at what was introduce that caused that.

Also you need to provide something for people to look at. Setup a codepen example and show the issue. https://codepen.io/brianvoe/pen/MWXqXXV

**Reply by sschausten** [reply]
Thanks for the reply!
I had tried to use the contentLocation, but it looks like I chose the wrong location. When I tried it now in codepen, I got the right location and the search now works in the modal dialogue.

**Reply by brianvoe** [reply]
Glad you figured it out.

Please consider donating. https://buymeacoffee.com/brianvoe
